URBNSwiftyConvenience/Classes/UIViewController+Extensions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  UIViewController+Extensions.swift
+//  Pods-URBNSwiftyConvenience_Example
+//
+//  Created by Bao Tran on 9/26/18.
+//
+
+import Foundation
+
+public extension UIViewController {
+    public func embedChildViewController(_ childViewController: UIViewController, insets: UIEdgeInsets = .zero,  safeAreaEdges: SafeAreaEdges = .none) {
+        addChildViewController(childViewController)
+        view.embed(subview: childViewController.view, insets: insets, safeAreaEdges: safeAreaEdges)
+        childViewController.didMove(toParentViewController: self)
+    }
+
+    public func removeChildViewController() {
```

Comment:
I’m not crazy about the name here. We’re removing _ourselves_ from the parent, not removing our child. We can’t call this `removeFromParentViewController` because that name is already taken.

Possibly `removeViewAndViewControl()`?